contrib/docker/dapp/dapp-start.sh

```diff
@@ -26,5 +25,5 @@ if [ -e $pidfile ]; then
 fi
 echo "starting dapp via node bundle"
 cd /home/doichain/dapp 
-nohup node build/bundle/main.js > dapp.log  2>&1 & echo $! > dapp.pid
```

Comment:
If we remove this, we neither get a dapp.log nor a dapp.pid 
which would be nice to have in my opinion.